test/mainnet-fork/Farm.t.sol

```diff
@@ -0,0 +1,176 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity >=0.8.0;
+
+import "./ForkTestBase.t.sol";
+
+interface IFarmLike {
+    function balanceOf(address account) external view returns (uint256);
+}
+
+contract MainnetControllerFarmTestBase is ForkTestBase {
+
+    address farm = 0x173e314C7635B45322cd8Cb14f44b312e079F3af;
```

Comment:
Add comment that this is the USDS SPK farm

Reply:
done